eslint-rules/lib/rules/typography-deprecation.js

```diff
@@ -47,11 +47,15 @@ module.exports = {
       if (source === importSource) {
         const specifiers = node.specifiers;
         if (specifiers) {
```

Comment:
if this code repeat itself maybe create a util function (in `utils.js`)

Reply:
Done